src/core/mavlink_mission_transfer.h

```diff
@@ -25,10 +25,10 @@ class Sender {
     Sender() = default;
     virtual ~Sender() = default;
     virtual bool send_message(mavlink_message_t& message) = 0;
-    virtual uint8_t get_own_system_id() const = 0;
-    virtual uint8_t get_own_component_id() const = 0;
-    virtual uint8_t get_system_id() const = 0;
-    virtual Autopilot autopilot() const = 0;
+    [[nodiscard]] virtual uint8_t get_own_system_id() const = 0;
```

Comment:
Just out of curiosity: shouldn't the compiler automatically warn when the return value of a const function is ignored? Is it just good practice to use both `const` and `[[nodiscard]]` then?